test/parallel/test-stream-construct.js

```diff
@@ -0,0 +1,216 @@
+'use strict';
```

Comment:
I may have missed, but is there a test for passing an error to the `_construct`/`construct` callback?

Reply:
now there is :)